### DISCUSSION TITLE
Auto ranges for IVT gearbox

### DISCUSSION THREAD

**Discussion by Tellto4** [Mogli12/KeyboardSteer#738] [Previous version]
I Mogly,
I am the son of a farmer and we own John Deere tractors. I play a bit during my free time farming and I noticed that the ranges of IVT gearboxes do not change automatically. However, on the tractors and harversters that I drive regularly, it turns out that the latter automatically switch from one to the other upwards (example: from 1 to 2 then 3 ...). In the event that the tractor slows down, the range downshifts either when it detects that the tractor is decelerating (a bit like a conventional powershift type gearbox but very low in rpm) or when it is there is a sudden change in load (still with a very slight jerk). I would have liked to help you directly on the GitHub but I do not control the coding enough. So I hope you will take this into account for your future VCA updates.

Congratulations for all your work and thank you for making us love farming so much thanks to this great mods.
Cordialy.

Tellto4

**Comment by Mogli12** [comment]
Hi @Tellto4 

Thanks for the nice feedback.

VCA cannot simulate the fixed ranges of the AutoPowr transmission. These ranges are used for a Fendt Vario transmission. The best option for a fully automatic IVT (JD AutoPowr, ZF, Class, Same, Deutz, ...) is the built in IVT transmission with single range only.

Best regards,
Stefan
